lib/protoparser/opentelemetry/pb/trace_fields.go

```diff
@@ -0,0 +1,68 @@
+package pb
+
+// trace_fields.go contains field names when storing OTLP trace span data in VictoriaLogs.
```

Comment:
Is it copied from somewhere? If so, can you mention it or add a link?

Reply:
This is defined by myself. Basically we need to have a field name for every attributes of a trace span (including resource, instrumentationscope, span) data.